### PR TITLE
Story: request judgement on challenge

### PR DESCRIPTION
**Pull request by raquellenn**
the storie of a judge judging a challenge

### CODE REVIEW

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl [hunk 1]
```diff
@@ -0,0 +1,31 @@
+domain ATQR
+
+context Challenges
+
+//função no Challenge
+
+💬 RequestJudgement? :challengeId
+  rules:
+    challenge.status = ChallengeStatus.Overdue
```
Comment (svallory) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: Did you mean to assign or compare?

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl [hunk 2]
```diff
@@ -0,0 +1,31 @@
+domain ATQR
+
+context Challenges
+
+//função no Challenge
```
Comment (svallory) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: This is represented by `root Challenge`

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+💬 RequestJudgement? :challengeId
+  rules:
+    challenge.status = ChallengeStatus.Overdue
+
+  sendJudgementRequestEmail? :challengeId
```
Comment (svallory) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: you cannot have a command inside another command

Comment (raquellenn) [reply libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: como assim? não posso ter um comando dentro de outro comando? mas a gente faz isso o tempo inteiro, não? haha

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl (hunk 1 above)
Comment (raquellenn) [reply libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: compare

Comment (svallory) [reply libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: Then you need `==` for it to be a comparison

File: libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl [hunk 4]
```diff
@@ -4,29 +4,22 @@ context Challenges
...
-      sendChallengeAccomplishedEmailToPlayer? :challengeId
-    when accomplished.false
-      @challengeJudgement = challengeJudgement.Failed
-      sendChallengeFailedEmailToPlayer? :challengeId
+    @challengeJudgement = ChallengeJudgement.Judged
   else
-    wait 1 day to
+    wait x day to
     sendJudgementRequestEmail? :challengeId
     if not answer
```
Comment (svallory) [inline libs/atqr/domain/src/lib/domain-spec/stories/judgeChallenge.disl]: you can't have a method run over days. You'll need to react to the `ANewDayArrived` event and count the days passed there

**Review by svallory — CHANGES_REQUESTED** [review]
Please also fix the title of the PR and add a description

**Comment by svallory** [comment]
Merged via #69

**Pull request closed without merging**